src/sbvr-api/permissions.ts

```diff
@@ -1670,6 +1698,10 @@ export const config = {
 					ALTER TABLE "role-has-permission"
 					ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
 				`,
+				'14.41.0-modified-at': `
```

Comment:
@thgreasi @Page- Do we need this pinned to latest release?

Reply:
@nitishagar probably yes and also `modified-at` doesn't sound descriptive of what the migration does. Maybe
```suggestion
				'14.42.0-api-key-expiry-date': `
```

Reply:
Should we be doing `..-modified-at` like other changes above or `..api-key-expiry-date`?

Reply:
We should call it `...-api-key-expiry-date`. The above two the say `modified at` are indeed migrations about that field.